src/terminal/SixelParser.h

```diff
@@ -193,12 +193,9 @@ class SixelImageBuilder: public SixelParser::Events
     Buffer buffer_; /// RGBA buffer
     CellLocation sixelCursor_;
     unsigned currentColor_;
-    struct
-    {
-        int nominator;
-        int denominator;
-    } aspectRatio_;
     bool explicitSize_ = false;
+    int aspectRatio_;
+    int sixelBandHeight_;
```

Comment:
just by looking at the uses of these two variables, at least the first one deserves some comment for better understanding that - it's an int rather a floating point and it's used as a count rather than a ratio. I'm just trying to understand that myself right now, so I was hoping for some little hint. :)
Also, both variables can never go negative, so maybe use `unsigned` instead?